### PR TITLE
Create Feed component

### PR DESCRIPTION
**Pull request by viettrung9012**
Issue reference: #8

### CODE REVIEW

File: docs/examples/views/FeedExample/index.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import ContentDateExample from './ContentDate.example';
+import ContentDateShorthandExample from './ContentDateShorthand.example';
+import SummaryDateExample from './SummaryDate.example';
+import SummaryDateShorthandExample from './SummaryDateShorthand.example';
+import ExtraImagesExample from './ExtraImages.example';
+import ExtraImagesShorthandExample from './ExtraImagesShorthand.example';
+import ExtraTextExample from './ExtraText.example';
+import ExtraTextShorthandExample from './ExtraTextShorthand.example';
+import SizeSmallExample from './SizeSmall.example';
+import SizeLargeExample from './SizeLarge.example';
```
Comment (mariolamacchia) [inline docs/examples/views/FeedExample/index.js]: Can we have these ordered alphabetically?

File: docs/examples/views/index.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export { default as CommentExample } from './CommentExample';
 export { default as StatisticExample } from './StatisticExample';
 export { default as ItemExample } from './ItemExample';
+export { default as FeedExample } from './FeedExample';
```
Comment (mariolamacchia) [inline docs/examples/views/index.js]: Can we have these ordered alphabetically?

File: src/views/Feed/FeedEvent.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      type: Array,
+      description: 'Shorthand for SuiFeedExtra with images'
+    },
+    extraText: {
+      type: String,
+      description: 'Shorthand for SuiFeedExtra with text'
+    },
+    meta: {
+      type: String,
+      description: 'Shorthand for SuiFeedMeta'
```
Comment (mariolamacchia) [inline src/views/Feed/FeedEvent.jsx]: Thanks for the descriptions!

File: src/views/Feed/FeedEvent.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    return (
+      <ElementType
+        {...getChildProps(this)}
+        class={classes(
+          'event',
+        )}
+      >
+        {this.image && <FeedLabel image={this.image} />}
+        {this.icon && <FeedLabel icon={this.icon} />}
+        {(this.content || this.date || this.summary || this.extraImages || this.extraText || this.meta) &&
```
Comment (mariolamacchia) [inline src/views/Feed/FeedEvent.jsx]: I'd move this condition to a function

File: src/views/index.js [hunk 5]
```diff
@@ -2,3 +2,4 @@ export * from './Card';
 export * from './Comment';
 export * from './Statistic';
 export * from './Item';
+export * from './Feed';
```
Comment (mariolamacchia) [inline src/views/index.js]: Can we have these ordered alphabetically?

File: docs/Component.vue [hunk 6]
```diff
@@ -160,6 +160,10 @@ export default {
             return { name, type: value.name };
           }
 
+          if (value.type instanceof Array) {
+            return { ...value, name, type: value.type.map(type => type.name).join('|') };
+          }
```
[the remark this answers is not in the document]
Comment (viettrung9012) [reply docs/Component.vue]: If a prop has two types e.g. type: [Boolean, Array] then it'll be displayed as Boolean | Array, previously the type field would be blank

**Comment by viettrung9012** [comment]
Updated PR according to review comments

**Pull request merged**